doc/smocodesample/net40app/Scripting/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Microsoft.SqlServer.Management.Smo;
+using Microsoft.SqlServer.Management.Sdk.Sfc;
+
+namespace Microsoft.SqlServer.Management.SmoSdkSamples
+{
+    // This application demostrates how to script out a sample database without dependencies and iterate through the list to display the results.
```

Comment:
this is still an issue.

---
In reply to: [88115682](https://github.com/Microsoft/sqltoolsservice/pull/130#discussion_r88115682) [](ancestors = 88115682)